language/move-lang/src/expansion/translate.rs

```diff
@@ -618,8 +627,57 @@ fn spec_member(context: &mut Context, sp!(loc, pm): P::SpecBlockMember) -> E::Sp
                 type_: t,
             }
         }
+        PM::Include {
+            name: pn,
+            type_arguments: ptys_opt,
+        } => {
+            let name = module_access(context, pn)?;
+            let type_arguments = optional_types(context, ptys_opt);
+            EM::Include {
+                name,
+                type_arguments,
+            }
+        }
+        PM::Apply {
+            name: pn,
+            type_arguments: ptys_opt,
+            patterns: ppatterns,
+        } => {
+            let name = module_access(context, pn)?;
+            let type_arguments = optional_types(context, ptys_opt);
+            let patterns = ppatterns
+                .into_iter()
+                .map(|p| function_pattern(context, p))
+                .collect();
+            EM::Apply {
+                name,
+                type_arguments,
+                patterns,
+            }
+        }
+        PM::Pragma { properties } => EM::Pragma { properties },
     };
-    sp(loc, em)
+    Some(sp(loc, em))
+}
+
+fn function_pattern(
```

Comment:
be sure to rename this if you rename the ast nodes

Reply:
Done, also in syntax.rs